index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
@chrislindor Again, right off the bat I see excellent indentation, and two spaces at that! Perfect :)